app/controllers/admin/settings_controller.rb

```diff
@@ -0,0 +1,40 @@
+# Admin controller for CMS settings
+class Admin::SettingsController < AdminController
+  # Render site settings form
+  def index
+    @settings = Setting.order( :name )
+  end
+
+  # New setting form submitted
+  def create
+    @setting = Setting.new( setting_params )
+
+    if @setting.save
+      flash[ :notice ] = 'New setting added'
```

Comment:
If the CMS ought to support i18n in the near future, consider adding any new flash messages to `config/locales/en.yml` and use `I18n.t(...)` in the controllers.